### DISCUSSION TITLE
The form grid layout lacks controllability

### DISCUSSION THREAD

**Discussion by gwsbhqt** [alibaba/formily#1813] [Q&A 2.x] [answered]
Thinking about this, I may need to hide or show certain fields in the grid layout

![image](https://user-images.githubusercontent.com/16471930/125916804-9f1af742-384f-4a27-9caf-408b8209ed62.png)

![image](https://user-images.githubusercontent.com/16471930/125917058-367dcfbf-1eef-4698-8b4c-6bb269e7023d.png)

Everything is normal at the moment, but...

![image](https://user-images.githubusercontent.com/16471930/125917228-175f3a84-78ef-447b-b3c7-34029f56c20c.png)

The content disappeared, but the placeholder did not disappear, and the grid column did not provide the corresponding controllability attribute

**Comment by janryWang** [comment, marked as answer]
你的gridSpan都是1，为啥要显示使用GridColumn

**Reply by gwsbhqt** [reply]
Sorry, it's my fault.
